sdk/tables/azure-tables/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "@azure/tables",
+  "version": "1.0.0-preview.1",
+  "description": "An isomorphic client library for the Azure Tables service.",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "browser": {
+    "stream": "./node_modules/stream-browserify/index.js",
```

Comment:
is this needed/used or leftover from the template?

Reply:
No they were just leftovers from the template. I have removed them now